Doc/Tutorial.tex

```diff
@@ -87,7 +87,7 @@
 %(hopefully this isn't going to be too wide for most people)
 \begin{rawhtml}
 <P ALIGN="center">
-<IMG ALIGN="center" SRC="images/biopython.jpg" TITLE="Biopython Logo" ALT="[Biopython Logo]" width="1024" height="288" />
+<IMG ALIGN="center" SRC="images/biopython_logo.svg" TITLE="Biopython Logo" ALT="[Biopython Logo]" width="1024" height="288" />
```

Comment:
This should probably be something like ``width="450" height="300"`` to respect the new logo aspect ratio (although Chrome does not stretch the image as I thought it might).